packages/expo-updates/ios/EXUpdates.podspec

```diff
@@ -55,8 +55,8 @@ Pod::Spec.new do |s|
     'GCC_TREAT_IMPLICIT_FUNCTION_DECLARATIONS_AS_ERRORS' => 'YES',
     'DEFINES_MODULE' => 'YES',
     'SWIFT_COMPILATION_MODE' => 'wholemodule',
-    'OTHER_CFLAGS[config=Debug]' => other_c_flags,
-    'OTHER_SWIFT_FLAGS[config=Debug]' => other_swift_flags
+    'OTHER_CFLAGS[config=*Debug*]' => other_c_flags,
+    'OTHER_SWIFT_FLAGS[config=*Debug*]' => other_swift_flags
```

Comment:
Does this same change need to be made to dev launcher? https://github.com/expo/expo/blob/main/packages/expo-dev-launcher/expo-dev-launcher.podspec#L75-L76

Reply:
Yeah it would be great to do the same there as well because I am not getting network inspection either. Feel free to adjust other places as well and update this PR.